src/index.js

```diff
@@ -61,7 +62,12 @@ class App extends Component<*> {
 
   hasSettingsChanged = (a, b) => a.settings !== b.settings;
 
-  hasAccountsChanged = (a, b) => a.accounts !== b.accounts;
+  hasAccountsChanged = (a, b) => {
+    return b.accounts.active.some(
+      account =>
+        !isEqual(account, a.accounts.active.find(ab => ab.id === account.id)),
```

Comment:
i don't agree with the changes done here. isEqual feels very complex in performance cost. why are we doing this exactly?

Reply:
the DB is updated everytime the app updates so we're actually writing in DB at each navigation action or anything that triggers an update on the store we should really move this kind of logic into redux directly imo